the-experts-fe/src/components/contractor/contractor.css

```diff
@@ -0,0 +1,37 @@
+.card-transition:hover {
+  transition: 0.1s;
+  transform: scale3d(1, 1.1, 1.1);
+}
+
```

Comment:
Try using a code formatter tool like Prettier to tidy up these unnecessary extra lines and spaces. It makes the code much easier to read!